otcextensions/sdk/tms/v1/resource_tag.py

```diff
@@ -0,0 +1,67 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from openstack import resource
+from openstack import exceptions
+
+
+class ResourceTag(resource.Resource):
+    resource_key = 'resource_tag'
+    base_path = '/resource-tags/batch-delete'
```

Comment:
maybe better to left only /resource-tags/ and append batch-delete or batch-create inside proxy?

Reply:
or even inside create/delete methods